packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -47,6 +47,7 @@ class StrapiApp {
     this.admin = {
       injectionZones,
     };
+    this.customFields = customFields;
```

Comment:
It's just we have to access custom fields with `app.customFields.customFields`. I'm not a big fan. I don't think we specified how to get all custom fields in the RFC but maybe the customFields object should be private and we should have a getter for that to avoid this weird path.

Reply:
I agree it seems like a good use case for a getter. But I implemented it this way to make it consistent with the other sibling files (Components, Fields, Middlewares and Reducers).

@alexandrebodin could you tell us what you think is best here?